src/parser/jobs/mnk/modules/TwinSnakes.tsx

```diff
@@ -82,45 +88,33 @@ export default class TwinSnakes extends Module {
 			return
 		}
 
-		switch (action.id) {
-		// Ignore TS itself, plus Form Shift, Anatman, and Meditation
-		// Should we double count 6SS?
-		// We use gcdsSinceTS because we don't want to double count FPF
-		case (this.data.actions.TWIN_SNAKES.id):
-			if (this.twinSnake && !this.twinSnake.end && this.gcdsSinceTS < TWIN_SNAKES_CYCLE_LENGTH) {
-				this.earlySnakes++
-			}
+		// Ignore FS and Meditation
+		if (TWIN_IGNORED_GCDS.includes(action.id)) { return }
 
-		// Count FPF, but check if it's a bad one
-		case (this.data.actions.FOUR_POINT_FURY.id):
-			if (!this.combatants.selected.hasStatus(this.data.statuses.TWIN_SNAKES.id)) {
-				this.failedFury++
+		if (!this.combatants.selected.hasStatus(this.data.statuses.TWIN_SNAKES.id)) {
+			if (action.id === this.data.actions.ANATMAN.id) {
+				this.failedAnts++
 			}
 
-		// Ignore Form Shift, for forced downtime can expect Anatman, or it'll just drop anyway
-		case (this.data.actions.FORM_SHIFT.id):
-			break
-
-		// Ignore Meditation, it's not really a GCD even tho it kinda is
-		case (this.data.actions.MEDITATION.id):
-			break
-
-		// Ignore Antman, but check if it's a bad one
-		case (this.data.actions.ANATMAN.id):
-			if (!this.combatants.selected.hasStatus(this.data.statuses.TWIN_SNAKES.id)) {
-				this.failedAnts++
+			if (action.id === this.data.actions.FOUR_POINT_FURY.id) {
+				this.failedFury++
 			}
 
-			break
+			return
```

Comment:
Do you intend to _not_ count either Four Point Fury or Anatman as GCDs in between twin snakes usages?  My understanding from our conversation on Discord was that FPF was supposed to be counted as a GCD.